about/index.html

```diff
@@ -91,14 +91,9 @@
       </div>
       <div class='col-md-4'>
         <h3>Minions</h3>
-        <p>Every year the minions help us run the show on the day.</p>
-        <ul class='info-list'>
-          <li>Ben Meier</li>
-          <li>Siobhan O'Donovan</li>
-          <li>Calvin Brizzi</li>
-          <li>Julia Lang</li>
-          <li>Jacques Heunis</li>
-        </ul>
+        <p>Every year the minions help us run the show on the day. Ben, Siobhan,
+          Calvin, Julia, Jacques, Sibulele, Shaheen, Ednecia - thank you!
+        </p>
```

Comment:
Yeah, sorry, I'd already made the branch and done a commit when I remembered the other changes. Sue me.